collector/interface.go

```diff
@@ -11,11 +12,20 @@ import (
 )
 
 var (
-	ifaceSubsystem = "interface"
+	ifaceSubsystem   = "interface"
+	ifaceErrors      = []error{}
+	totalIfaceErrors = 0.0
+	ifaceDescrKeys   []string
+)
+
+func getInterfaceDesc() map[string]*prometheus.Desc {
+	var ifacePhysicalLabels = []string{"interface"}
+	if len(ifaceDescrKeys) > 0 {
```

Comment:
This simply adds all of the configured interface description keys to include in the metrics to the metric definition